frontend/src/views/CellCards/components/CellCard/components/Description/index.tsx

```diff
@@ -30,10 +32,23 @@ export default function Description({
     else setDescriptionCl("");
   }, [rawDescriptionGpt, rawDescriptionCl]);
 
+  const copyHandler = () => {
+    if (window) {
```

Comment:
nit: 

prefer guard clause to prevent nesting!

E.g.,

```ts
if (!window) return;

const selectedText = window.getSelection()?.toString().trim();

if (selectedText !== "") {
  track(EVENTS.CG_COPY_CELL_TYPE_DESCRIPTION);
}
```